package.json

```diff
@@ -22,8 +22,8 @@
     "klona": "^2.0.6"
   },
   "devDependencies": {
-    "@digitalbazaar/data-integrity": "^1.3.1",
-    "@digitalbazaar/data-integrity-context": "^1.0.0",
+    "@digitalbazaar/data-integrity": "digitalbazaar/data-integrity#update-context",
+    "@digitalbazaar/data-integrity-context": "digitalbazaar/data-integrity-context#drop-support-for-node-16",
```

Comment:
Guessing these will change once upstream PR work is complete?

Reply:
@BigBlueHat Yep, they've been updated now.